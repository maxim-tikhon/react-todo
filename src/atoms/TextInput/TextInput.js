import styles from "./TextInput.module.scss";

const TextInput = ({className, ...props}) => {
  return (
    <input
      type="text"
      {...props}
      className={`${styles.textInput} ${className}`}
    />
  );
}

export default TextInput;
