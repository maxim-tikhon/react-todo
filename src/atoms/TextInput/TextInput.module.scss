.textInput {
  font-size: 16px;
  border: none;
  outline: none;
//   margin-left: 10px;
 // width: calc(100% - 2rem);
}
