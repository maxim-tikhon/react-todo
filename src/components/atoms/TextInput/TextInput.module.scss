.textInput {
  font-size: 16px;
  border: none;
  outline: none;
}
