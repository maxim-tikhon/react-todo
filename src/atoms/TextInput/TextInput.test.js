import renderer from 'react-test-renderer';
import TextInput from './TextInput';

describe('TextInput', () => {
	it('renders correctly', () => {
		const output = renderer.create(<TextInput placeholder="Test" />).toJSON();
		expect(output).toMatchSnapshot();
	});
});
