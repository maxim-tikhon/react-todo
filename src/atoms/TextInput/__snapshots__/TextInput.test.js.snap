// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextInput renders correctly 1`] = `
<input
  className="textInput undefined"
  onKeyDown={[Function]}
  placeholder="Test"
  type="text"
/>
`;
